app/grandchallenge/archives/views.py

```diff
@@ -716,6 +716,9 @@ def object(self):
     def base_object(self):
         return Archive.objects.get(slug=self.kwargs["slug"])
 
+    def get_interface_viewname(self):
+        return "components:component-interface-list-archives"
```

Comment:
I think this would make more sense as a model property on the base model. Then it you also don't need to pass it to the form, but can just just call `base_object.interface_viewname` there.

Reply:
I agree, done.